docs/schema/validation.json

```diff
@@ -43,7 +43,7 @@
     },
     "absolute_links": {
       "title": "warning level when absolute links are used",
-      "$ref": "#/$defs/warning_levels"
+      "enum": ["warn", "info", "ignore", "relative_to_docs"]
```

Comment:
Thanks for the PR. Could we please keep the definition reference, and add the `relative_to_docs` as another option, possibly by using `oneOf` or similar? [This would also be in line with how MkDocs implements it](https://github.com/mkdocs/mkdocs/blob/7e4892ab2dd4a52efd95a9de407eee63310e0780/mkdocs/config/defaults.py#L28-L32).

Reply:
Thanks; my search of the JSON schema docs was not thorough enough to find `oneOf`. 

I've pushed a new commit.